packages/angular/src/generators/ng-add/migrate-from-angular-cli.ts

```diff
@@ -36,6 +38,10 @@ export async function migrateFromAngularCli(
   const projects = getAllProjects(tree);
   const options = normalizeOptions(tree, rawOptions, projects);
 
+  const defaultProject = projects.apps.find((app) =>
+    app.config.sourceRoot?.startsWith('src')
```

Comment:
nit: I think it's more intuitive to assert whether the project is in the root of the Angular CLI workspace. That's the check done inside the app migrator for other tasks.
```suggestion
    app.config.root === ''
```